default-jammy.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Sorry, I meant move this into `CMakePresets.json` so that one can do
```
cmake --presets default-jammy
```
to run it.

We can then add more presents into `CMakePresets.json` and user can add their own presets into the user file.